framework/src/meshgenerators/ExtraNodesetGenerator.C

```diff
@@ -91,65 +80,67 @@ ExtraNodesetGenerator::generate()
   locator->enable_out_of_mesh_mode();
 
   const auto tolerance = getParam<Real>("tolerance");
-  for (const auto & c : coord)
-  {
-    Point p;
-    if (c.size() < dim)
-      paramError("coord",
-                 "Coordinate ",
-                 Moose::stringify(c),
-                 " does not have enough components for a ",
-                 dim,
-                 "D mesh.");
-
-    if (c.size() > 3)
-      paramError("coord",
-                 "Coordinate ",
-                 Moose::stringify(c),
-                 " has too many components. Did you maybe forget to separate multiple coordinates "
-                 "with a ';'?");
-
-    for (unsigned int j = 0; j < c.size(); ++j)
-      p(j) = c[j];
-
-    // locate candidate element
-    bool on_node = false;
-    bool found_elem = false;
-    const Elem * elem = (*locator)(p);
-    if (elem)
+  if (_pars.isParamValid("coord"))
```

Comment:
same here. Having an empty coord would also make the loop below do nothing.

Reply:
This I agree but the `isParamValid("nodes")` is needed